features-json/lazyload.json

```diff
@@ -1,16 +1,12 @@
 {
   "title":"Resource Hints: Lazyload",
-  "description":"Gives a hint to the browser to lower the loading priority of a resource.",
+  "description":"Gives a hint to the browser to lower the loading priority of a resource. Please note that this is a legacy attribute, see the [`loading`](https://caniuse.com/#feat=loading-lazy-attr) attribute for the new standardized API.",
```

Comment:
@Fyrd Since I [added a note](https://github.com/Fyrd/caniuse/pull/4928/files#diff-c5441aae512c0f4a638011b54ed30267R340), is it overkill to have a warning in the feature description aswell?

Reply:
I'd suggest removing it from the notes and leaving it in the description, since folks are more likely to notice it there.

Reply:
Removed note in  https://github.com/Fyrd/caniuse/pull/4928/commits/e68eb84b46e67ba87d8d32d075f7116b254c95f4. Thanks! :)